chia/daemon/server.py

```diff
@@ -1084,9 +1085,15 @@ async def start_service(self, request: Dict[str, Any]):
                 self.services.pop(service_command)
                 error = None
             else:
-                error = f"Service {service_command} already running"
-
-        if error is None:
+                self.log.info(f"Service {service_command} already running")
+                already_running = True
```

Comment:
Can't the removal of `error =` here affect behavior in existing cases?

Reply:
I checked and this change doesn't impact any of the callers that I'm aware of (`chia start <service>` and the GUI). If you run `chia start wallet` when the wallet is already running, the behavior is as-expected: "chia_wallet: Already running, use '-r' to restart"

Had I left the error intact, GUI code would need to be changed to handle the case where an error is returned but the service is already running... I felt that would be an uglier solution.